src/ganttChartSettingsModels.ts

```diff
@@ -259,61 +264,95 @@ export class LegendCardSettings extends Card {
         value: { value: "#000000" }
     });
 
-    fontSize = new formattingSettings.NumUpDown({
-        name: "fontSize",
-        displayNameKey: "Visual_TextSize",
-        value: FontSizeSettings.MinFontSize,
-        options: {
-            minValue: {
-                type: powerbiVisualsApi.visuals.ValidatorType.Min,
-                value: FontSizeSettings.MinFontSize,
+    font = new formattingSettings.FontControl({
+        name: "font",
+        displayNameKey: "Visual_Font",
+        fontSize: new formattingSettings.NumUpDown({
+            name: "fontSize",
+            value: FontSizeSettings.MinFontSize,
+            options: {
+                minValue: { value: 5, type: ValidatorType.Min },
```

Comment:
I assume it would be correct to add a new value for FontSizeSettings. For example some default value, it will be 8 like the MinFontSize now, and change Min to 5. Check it's usage in other places accordingly.